src/v2/Apps/Onboarding/Components/OnboardingOrderedSet.tsx

```diff
@@ -0,0 +1,111 @@
+import { Column, GridColumns, Box } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { OnboardingLoadingCollection } from "./OnboardingLoadingCollection"
+import { OnboardingOrderedSet_orderedSet } from "v2/__generated__/OnboardingOrderedSet_orderedSet.graphql"
+import { OnboardingOrderedSetQuery } from "v2/__generated__/OnboardingOrderedSetQuery.graphql"
+import { FC } from "react"
+// import { EntityHeaderArtistFragmentContainer } from "v2/Components/EntityHeaders/EntityHeaderArtist"
+// import { extractNodes } from "v2/Utils/extractNodes"
+
+interface OnboardingOrderedSetProps {
+  orderedSet: OnboardingOrderedSet_orderedSet
+}
+
+export const OnboardingOrderedSet: FC<OnboardingOrderedSetProps> = ({
+  orderedSet,
+}) => {
+  // const artists = extractNodes(orderedSet)
+  // TODO: artists is currently an empty array will fix
+
+  return (
+    <>
+      <GridColumns>
+        <Column span={6}>
+          <Box
+            color="black100"
+            border="1px solid"
+            width="100%"
+            height="100%"
+          ></Box>
+        </Column>
+        <Column span={6}>
+          {/* {artists.map(artist => {
+            if (!artist || artist.name) return null
+
+            return (
+              <EntityHeaderArtistFragmentContainer
+                artist={artist}
+                key={artist.internalID}
+              />
+            )
+          })} */}
+        </Column>
+      </GridColumns>
+    </>
+  )
+}
+
+const OnboardingOrderedSetFragmentContainer = createFragmentContainer(
+  OnboardingOrderedSet,
+  {
+    orderedSet: graphql`
+      fragment OnboardingOrderedSet_orderedSet on OrderedSet {
+        orderedItemsConnection(first: 50) {
+          edges {
+            node {
+              ... on Artist {
+                name
+                slug
+                internalID
+                image {
+                  imageURL
+                  internalID
+                }
+                formattedNationalityAndBirthday
+              }
```

Comment:
Use the fragment

Reply:
right... gotta spread those fields into the query so they're available on the Artist obj right?

Reply:
Yep: to use any fragment container you need to include that specific associated fragment